airflow/providers/google/cloud/example_dags/example_gcs_to_bigquery.py

```diff
@@ -53,7 +53,7 @@
 
 delete_test_dataset = BashOperator(
     task_id='delete_airflow_test_dataset',
-    bash_command='bq rm -rf airflow_test',
+    bash_command='bq rm -r -f airflow_test',
```

Comment:
Here is the problem with test isolation. When two tests are run simultaneously, they will conflict. Can you move the dataset name to the environment variables? We want to generate environment variables to ensure isolation between tests.

Reply:
Probably a newbie question, but how do environment variables isolate the test, compared to the hardcoded command that's in there right now? Thanks in advance @mik-laj   , Will adjust it tomorrow

Reply:
We generate environment variables for each test run.

To set environment variables we have a script that is similar to the following.
```bash
if [[ ! -f "${RANDOM_FILE}" ]]; then
    echo "${RANDOM}" > "${RANDOM_FILE}"
fi

RANDOM_POSTFIX=$(cat "${RANDOM_FILE}")
AIRFLOW_BREEZE_SHORT_SHA="${AIRFLOW_BREEZE_SHORT_SHA:="build"}"
AIRFLOW_BREEZE_TEST_SUITE="${AIRFLOW_BREEZE_TEST_SUITE:="test"}"
AIRFLOW_BREEZE_UNIQUE_SUFFIX=${AIRFLOW_BREEZE_TEST_SUITE}-${AIRFLOW_BREEZE_SHORT_SHA}-${RANDOM_POSTFIX}
GCP_FIRESTORE_DATASET_NAME=test_firestore_to_bigquery_${RANDOM_POSTFIX}
```
This script generates unique resource names for each CI launch. This script generates unique resource names for each CI launch.

During development, to make sure everything works, I often run tests using the following command.
```
GCP_GCS_BUCKET=airflow-life-science-$RANDOM pytest tests/providers/google/cloud/operators/test_life_sciences_system.py --system google -s
````
That way I can be sure that everything works and I don't have side effects from another run.

Reply:
Interesting. So if I create the environment variables in the file like this.
`DATASET = os.environ.get("GCP_DATASET_NAME", 'airflow_test')` 
They will be picked up and adjusted to a unique name by the script?

Reply:
Yes. Exactly. Each test run has its own runtime environment defined by ... environment variables. It is not yet complete in the community, but it will work like that. At [Polidea](polidea.com), we launched system tests on CI in this way.

Reply:
thanks. appreciate the clear explanation